### ISSUE TITLE
Exception on toString in Chrome 49

### ISSUE THREAD

**Issue by glenjamin** [ljharb/is-typed-array#3]
When using (current) Chrome Version 49.0.2623.87 (64-bit)

Error is
```
Uncaught TypeError: Cannot read property 'get' of undefined(anonymous function) @ index.js:27
```

Which maps to: https://github.com/ljharb/is-typed-array/blob/master/index.js#L27

Having a look in the console I can see that `gOPD(proto, Symbol.toStringTag)` is coming out as undefined.

This was previously working, it seems like Chrome has added some of the features which are detected, but not all the ones that this code expects?

**Comment by ljharb** [comment]
Thanks! I'll take a look.

**Comment by ljharb** [comment]
@glenjamin can you provide the code you're using to generate that error?

**Comment by ljharb** [comment]
nvm, figured it out. Thanks for the report!

**Comment by ljharb** [comment]
Released as v1.0.4 (also shortly releasing v1.0.1 of `which-typed-array` to fix the same problem)

**Comment by glenjamin** [comment]
Awesome, thanks for the quick fix.
